### ISSUE TITLE
Private  repository, public issue, need an entry link

### ISSUE THREAD

**Issue by mukaiu** [gogits/gogs#4356]
- Gogs version (or commit ref):  0.10.31.0327 / 0.11 RC 
- Git version: 
- Operating system: CentOS
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite

## Description

Private  repository, public issue. Can only be accessed via URL
The dashboard needs an entry link for easy access
...

**Comment by ModdyLP** [comment]
Please remove the Button "new Site" in the Wiki for Anonymous. They dont need this Button.

**Comment by Unknwon** [comment]
@ModdyLP "new Site"?

**Comment by ModdyLP** [comment]
![screenshot_20170406-183028](https://cloud.githubusercontent.com/assets/15385020/24765266/7400c8b8-1af7-11e7-9115-b1deb2923df4.png)

Here you see it, i am anonymous and see a link to create a new wiki site.

**Comment by Unknwon** [comment]
@ModdyLP Can you reproduce on demo site? I can't reproduce..

**Comment by ModdyLP** [comment]
No, there is no button. I am using the latest release, but not the develop version. May it was removed after the release in the develop version?

**Comment by Unknwon** [comment]
@ModdyLP define "latest release"

**Comment by ModdyLP** [comment]
Release 0.11, I see you release a day ago another release. I haven't seen it because normally the release are on Tuesday. :D I will update an look if there is a change.

**Comment by Unknwon** [comment]
@ModdyLP yeah... found a critical bug :trollface:

**Comment by Twisterat57** [comment]
Nice stuff... **just missing access link for a private repo with public wiki/issues XD**
But very happy to have this new feature on Gogs. Thanks guys.

**Comment by ModdyLP** [comment]
I have make the Update an voila the bug is already there. I have after the Update also the button for creating a new wiki site as anonymous.
